File: teamify_flutter/find.dart
import 'dart:io';

void main() {
  final dir = Directory('lib/screens/meeting');
  if (!dir.existsSync()) return;
  final out = File('find_output.txt').openWrite();
  for (final entity in dir.listSync(recursive: true)) {
    if (entity is File && entity.path.endsWith('.dart')) {
      final lines = entity.readAsLinesSync();
      for (int i = 0; i < lines.length; i++) {
        final lower = lines[i].toLowerCase();
        if (lower.contains('raise') || lower.contains('camera') || lower.contains('participants') || lower.contains('leave')) {
          out.writeln('${entity.path}:${i + 1}: ${lines[i]}');
        }
      }
    }
  }
  out.close();
}

File: teamify_flutter/find.js
const fs = require('fs');
const path = require('path');

function search(dir) {
    if (!fs.existsSync(dir)) return;
    const files = fs.readdirSync(dir);
    for (const file of files) {
        const fullPath = path.join(dir, file);
        if (fs.statSync(fullPath).isDirectory()) {
            search(fullPath);
        } else if (fullPath.endsWith('.dart')) {
            const lines = fs.readFileSync(fullPath, 'utf8').split('\n');
            for (let i = 0; i < lines.length; i++) {
                const lower = lines[i].toLowerCase();
                if (lower.includes('raise') || lower.includes('camera') || lower.includes('participants') || lower.includes('leave')) {
                    fs.appendFileSync('find_output.txt', `${fullPath}:${i + 1}: ${lines[i]}\n`);
                }
            }
        }
    }
}

fs.writeFileSync('find_output.txt', '');
search('lib/screens/meeting');
